Cargo.toml

```diff
@@ -170,6 +170,17 @@ overflow-checks = false
 panic = 'unwind'
 rpath = false
 
+[profile.ci]
+inherits = "dev"
+incremental = false
+
+# ci turns off debug info, etc for dependencies to allow for smaller binaries making caching more effective
+[profile.ci.package."*"]
+debug = false
```

Comment:
We also found this setting particularly effective for speeding up our CI in influxdb_iox: https://doc.rust-lang.org/cargo/reference/profiles.html#debug